torchdata/datapipes/iter/load/huggingface.py

```diff
@@ -62,8 +57,6 @@ class HuggingFaceHubReaderIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
 
     """
 
-    source_datapipe: IterDataPipe[str]
```

Comment:
Interesting how this was never used but we didn't notice... 

Thanks for spotting this!